src/screens/SettingsScreen.js

```diff
@@ -52,6 +58,8 @@ const INITIAL_FILTER = [
   { id: TOP_FILTER.LIST_TYPES, title: texts.settingsTitles.tabs.listTypes, selected: false }
 ];
 
+const ONBOARDING_STORE_KEY = 'ONBOARDING_STORE_KEY';
```

Comment:
this key shouldn't be duplicated. it is already existing in `OnboardingManager`, where we should export it to use it here.

Reply:
a7587b4e5e398ac83c0cd01a398797166b4d72b2